meos/src/general/pg_types.c

```diff
@@ -70,6 +72,8 @@
   extern Datum timestamp_out(PG_FUNCTION_ARGS);
   extern Datum timestamptz_out(PG_FUNCTION_ARGS);
   extern Datum interval_out(PG_FUNCTION_ARGS);
+  extern Datum range_out(PG_FUNCTION_ARGS);
+  extern Datum multirange_out(PG_FUNCTION_ARGS);
```

Comment:
This should be removed too, since it's now in `mobilitydb/src.general/type_util.c`